src/core/face_engine_haar_lbph.py

```
@@ -2,6 +2,41 @@
+    if w > h:
+        img = cv2.rotate(img, cv2.ROTATE_90_CLOCKWISE)
+    return img
+
+def load_and_correct_orientation(image_path_or_array):
+    # Accepts either a file path or a numpy array (BGR)
+    if isinstance(image_path_or_array, str):
+        img = Image.open(image_path_or_array)
+    else:
+        img = Image.fromarray(cv2.cvtColor(image_path_or_array, cv2.COLOR_BGR2RGB))
+    try:
+        for orientation in ExifTags.TAGS.keys():
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)